extensions/blocks/podcast-player/components/playlist.js

```diff
@@ -20,11 +20,13 @@ const TrackIcon = ( { isPlaying, isError, className } ) => {
 
 	if ( isError ) {
 		name = 'error';
-		/* translators: This is text to describe the current state. This will go before the track title, such as "Error: [The title of the track]" */
+		/* translators: This is text to describe the current state. This will go
```

Comment:
should we replace this by

```ec6
/*
 *
 */
```
?

```suggestion
		/*
		 * translators: This is text to describe the current state. This will go
		 * before the track title, such as "Error: [The title of the track]".
		 */
```

Reply:
I'm not 100% sure. What if the translators comment requires that format? These comments are being programmatically extracted from the source code so we need to make sure we don't break that.

Reply:
We cannot do that, unfortunately. Please see https://github.com/Automattic/jetpack/pull/15281#discussion_r404142106 and https://github.com/Automattic/jetpack/pull/15281#discussion_r404142106 for more details on that!